inc/microbit/mpconfigport.h

```diff
@@ -80,10 +80,10 @@ void mp_hal_stdout_tx_strn_cooked(const char *str, mp_uint_t len);
 
 // extra built in names to add to the global namespace
 extern const struct _mp_obj_fun_builtin_t mp_builtin_help_obj;
-extern const struct _mp_obj_fun_builtin_t mp_builtin_open_obj;
+extern const struct _mp_obj_fun_builtin_t microbit_open_obj;
```

Comment:
I don't see why this needs to be renamed.  It's still the builtin open function (and doesn't need to be microbit specific, it could eventually support opening of other kinds of files).